typescript/packages/common-charm/src/syncRecipe.ts

```diff
@@ -8,6 +8,8 @@ import {
 } from "@commontools/runner";
 import { buildRecipe } from "./localBuild.js";
 
+// FIXME(jake): This needs to be settable by environment variable...
+// If this is hardcoded, then it is not possible to develop spellbook locally.
 export const BLOBBY_SERVER_URL = "https://toolshed.saga-castor.ts.net/api/storage/blobby";
```

Comment:
We expose /api/storage/blobby via VITE - so we should be able to use the url of /api/storage/blobby?

```
      "/api/storage/blobby": {
        target: process.env.TOOLSHED_API_URL ?? "http://localhost:8000/",
        changeOrigin: true,
      },
```

Reply:
oh, but this might be run without VITE ... hence the fixme...